ROADMAP.md

```diff
@@ -0,0 +1,23 @@
+# Roadmap
+
+## Top-level goals
+
+* Extensibility: custom components, shaders
+* Performance: raycasting (three.js), head tracking on mobile (webvr-polyfill), anything DOM/Custom Elements-related
+* Ecosystem: enable discovery and distribution of assets, components, scenes, snippets, shaders
+
+## `0.2.0`
+
+* Merge aframe-core repo into aframe repo
+* Custom shaders
+* New primitives implementation (extending entities instead of templates of grouped elements)
+* Video playback issues (on Android, iOS)
+* Improved mobile experience (orientation lock, initial position, etc.)
+* Systems (shared logic of components)
+* Texture caching
+* Improved asset management and loading
+* obj model support
+
+## Future
+
+* glTF support
```

Comment:
I'd add physics, interactivity (grabbing things, moving places), multiplayer, and new WebVR API

I might deprioritize glTF and remove from the ROADMAP altogether. That's not something we control, and even when glTF comes out, it's not that important because OBJ is the much much more common format.